### ISSUE TITLE
Allow configuration for start/end of the day

### ISSUE THREAD

**Issue by LAlbertalli** [bear/parsedatetime#188]
Currently, the library considers the start of the day as 9AM and the end of the day as 5PM. Although these are sensible settings in some use case, they should be configurable, at least to allow midnight to midnight days, but possibly other values.

**Comment by idpaterson** [comment]
Thanks, see #186. We're definitely looking for feedback on how such a configuration should work.

**Comment by idpaterson** [comment]
Sorry this may not quite be a duplicate of the other issue... can reopen but I suspect that it will be easier to talk about this case along with the other configurations. I believe that here you are referring to the implicit start or end times that you get from a phrase like *tomorrow* rather than explicit phrases like *eod*.

**Comment by LAlbertalli** [comment]
@idpaterson Yes, I was referring to the implicit start of the day that I get from "today".  But the behavior is also hardwired for "end of" consider "end of week" being parsed as 02/09/2016 17:00.

**Comment by pbdeuchler** [comment]
Not sure if this or #186 is the place to put this, but I have a similar problem:

```python
>>> cal.parse("end of day")
(time.struct_time(tm_year=2016, tm_mon=9, tm_mday=2, tm_hour=17, tm_min=0, tm_sec=0, tm_wday=4, tm_yday=246, tm_isdst=1), 3)

>>> cal.parse("tomorrow end of day")
(time.struct_time(tm_year=2016, tm_mon=9, tm_mday=3, tm_hour=9, tm_min=0, tm_sec=0, tm_wday=5, tm_yday=247, tm_isdst=-1), 3)

>>> cal.parse("tomorrow start of day")
(time.struct_time(tm_year=2016, tm_mon=9, tm_mday=3, tm_hour=9, tm_min=0, tm_sec=0, tm_wday=5, tm_yday=247, tm_isdst=-1), 1)
```

As you can see, "tomorrow end of day" ends up being the same time as "tomorrow start of day" (9am), while a regular "end of day" implicates 5pm

**Comment by idpaterson** [comment]
@pbdeuchler that's a good observation, can you start a new issue for that please?

**Comment by pbdeuchler** [comment]
@idpaterson done: #189
